docs/UserGuide.md

```diff
@@ -41,13 +41,13 @@ If you encounter any issues in launching and using the app, feel free refer to t
 * Items in square brackets are optional.
   * e.g. `n/NAME [c/CLASS_NAME]` can be used as `n/John Doe c/CS1231S` or as `n/John Doe`.
 * Items with `...` after them can be used multiple times.
-  * e.g. `[c/CLASS_NAME...]` can be used as ` ` (i.e. 0 times since it is also optional), `c/CS1101S`, `c/CS2030 c/ST2334` etc.
+  * e.g. `c/CLASS_NAME...` can be used as `c/CS1101S`, `c/CS2030 c/ST2334`, etc.
```

Comment:
```suggestion
  * e.g. `c/CLASS_NAME...` can be used as `c/CS1101S` or `c/CS2030 c/ST2334`, etc.
```

Reply:
I think this is okay, since we're have an `etc` at the back?